digitalocean/loadbalancer/resource_loadbalancer.go

```diff
@@ -403,6 +403,11 @@ func resourceDigitalOceanLoadBalancerV0() *schema.Resource {
 					},
 				},
 			},
+			"type": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "the type of the load balancer (GLOBAL or REGIONAL)",
+			},
```

Comment:
Can an existing load balancer be changed from regional to global? If not, we probably want to set `ForceNew` on this.
```suggestion
			"type": {
				Type:        schema.TypeString,
				Optional:    true,
				ForceNew: true,
				Description: "the type of the load balancer (GLOBAL or REGIONAL)",
			},
```

Reply:
it cannot.